src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -51,280 +64,272 @@ public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l,
-                        var row = rows[i];
-                        var key = dom.QuerySelector("link[rel=alternate]").GetAttribute("href").Split('=').Last();
-
-                        var release = new ReleaseInfo();
                         var torrentTxt = row.QuerySelector(".torrent_txt, .torrent_txt2").QuerySelector("a");
                         //if (torrentTxt == null) continue;
-                        release.Title = torrentTxt.GetAttribute("title");
-                        var descr = row.QuerySelector("span")?.GetAttribute("title") + " " + row.QuerySelector("a.infolink")?.TextContent;
-                        release.Description = descr.Trim();
-
-                        release.MinimumRatio = 1;
-                        release.MinimumSeedTime = 172800; // 48 hours
-                        release.DownloadVolumeFactor = 0;
-                        release.UploadVolumeFactor = 1;
-
+                        var infoLink = row.QuerySelector("a.infolink");
+                        var imdbId = ParseUtil.GetLongFromString(infoLink?.GetAttribute("href"));
+                        var desc = row.QuerySelector("span")?.GetAttribute("title") + " " +
+                                   infoLink?.TextContent;
                         var downloadLink = SiteLink + torrentTxt.GetAttribute("href");
-                        var downloadId = downloadLink.Substring(downloadLink.IndexOf("&id=") + 4);
-
-                        release.Link = new Uri(SiteLink + "torrents.php?action=download&id=" + downloadId + "&key=" + key);
-                        release.Comments = new Uri(SiteLink + "torrents.php?action=details&id=" + downloadId);
-                        release.Guid = new Uri(release.Comments + "#comments");
-
-                        release.Seeders = ParseUtil.CoerceInt(row.QuerySelector(".box_s2").QuerySelector("a").TextContent);
-                        release.Peers = ParseUtil.CoerceInt(row.QuerySelector(".box_l2").QuerySelector("a").TextContent) + release.Seeders;
-                        var imdblink = row.QuerySelector("a[href*=\".imdb.com/title\"]")?.GetAttribute("href");
-                        if (!string.IsNullOrWhiteSpace(imdblink))
-                            release.Imdb = ParseUtil.GetLongFromString(imdblink);
+                        var downloadId = ParseUtil.GetArgumentFromQueryString(downloadLink, "id");
+
+                        //Build site links
+                        var baseLink = SiteLink + "torrents.php?action=download&id=" + downloadId;
+                        var commentsUri = new Uri(baseLink);
+                        var guidUri = new Uri(baseLink + "#comments");
+                        var linkUri = new Uri(QueryHelpers.AddQueryString(baseLink, "key", key));
+
+                        var seeders = ParseUtil.CoerceInt(row.QuerySelector(".box_s2").QuerySelector("a").TextContent);
+                        var leechers = ParseUtil.CoerceInt(row.QuerySelector(".box_l2").QuerySelector("a").TextContent);
```

Comment:
combine both selectors in 1?

Reply:
Not sure what you're asking? These are two different selector strings.

Reply:
row.QuerySelector(".box_l2 a")
but remember to test it, sometimes the selectors don't work as expected

Reply:
Tested and working